images/cfw/etc/init.d/S98webhmi

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          webhmi
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Starts the webhmi service
+# Description:       Runs the little_webhmi.py Python script as a service at boot.
+### END INIT INFO
+
+# Using the Python interpreter from the virtual environment:
+DAEMON_PATH="/opt/python/bin/"
+
+# Script to be executed:
+DAEMON="python3 /usr/etc/system/webhmi/little_webhmi.py"
```

Comment:
WebHMI needs to be part of x1plusd, not running in the background as its own python service